data-model-changes.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#(set -o igncr) 2>/dev/null
+set -o igncr; # this comment is required to trick cygwin into dealing with windows vs. linux EOL characters
+
+function helpFunction {
+   echo ""
+   echo "Connection options:"
+   echo -e "\t-h database server host or socket directory"
+   echo -e "\t-u database user name"
+   exit 1 # Exit script after printing help
+}
+
+while getopts 'h:u:?' flag; do
+    case "${flag}" in
+        h) HOST="${OPTARG}" ;;
+        u) USER="${OPTARG}" ;;
+        *) helpFunction ;;
+    esac
+done
+
+# Print helpFunction in case parameters are empty
+if [ -z "$HOST" ] || [ -z "$USER" ]
+then
+   echo "Some or all of the parameters are empty";
+   echo Host: $HOST
+   echo User: $USER
+   helpFunction
+fi
+
+#Exit on failure
+set -e
```

Comment:
Can you move this to above any commands that happen in the script? If we're bombing out on failures I'd like to make sure we do it as early as the failure happens